app/build.gradle

```diff
@@ -10,8 +10,8 @@ android {
         applicationId "com.cobresun.factfulnewsandroid"
         minSdkVersion 23
         targetSdkVersion 28
-        versionCode 1
-        versionName "1.0"
+        versionCode 3
+        versionName "1.2"
```

Comment:
Much better to have the version upgrades be separate PRs so we can see them in the commit log

Reply:
noted for future but for now this updated version has aldready been uploaded to play